misc/scripts/update.sh

```diff
@@ -56,24 +59,52 @@ wait && stty "$tty_settings"
 sudo chmod +x /bin/pacstall
 sudo chmod +x /usr/share/pacstall/scripts/*
 
+
+echo "$USERNAME $BRANCH" | sudo tee "$STGDIR/repo/update" > /dev/null
+
+new_pacstall_branch=( $(cat $STGDIR/repo/update) )
+new_pacstall_version=( $(pacstall -V) )
+
 # Bling Bling update ascii
+if [[ "${new_pacstall_branch[1]}" != "master" ]]; then
+echo '
+                                     ∩~-~∩
+ _____               _        _ _   ξ ･×･ ξ
+|  __ \             | |      | | |  ξ　~　ξ
+| |__) |_ _  ___ ___| |_ __ _| | |  ξ　　 ξ
+|  ___/ _` |/ __/ __| __/ _` | | |  ξ　　 “～～～〇
+| |  | (_| | (__\__ \ || (_| | | |  ξ　　 　　　 ξ
+|_|   \__,_|\___|___/\__\__,_|_|_|  ξ_ξ ξ_ξ ξ_ξξ_ξ
+'
+else
 echo '
     ____                  __        ____
    / __ \____ ___________/ /_____ _/ / /
   / /_/ / __ `/ ___/ ___/ __/ __ `/ / /
  / ____/ /_/ / /__(__  ) /_/ /_/ / / /
 /_/    \__,_/\___/____/\__/\__,_/_/_/
 '
-if [[ $USERNAME == "pacstall" ]] && [[ $BRANCH == "master" ]]; then
-	echo -e "[${BGreen}+${NC}] INFO: You are at version $(pacstall -V)"
-	echo -e "[${BYellow}*${NC}] WARNING: Be sure to check our GitHub release page to make sure you have no incompatible code: https://github.com/pacstall/pacstall/releases"
-else
-	echo -e "[${BYellow}*${NC}] WARNING: You are using a ${RED}development${NC} version of $(pacstall -V)"
-	echo -e "[${BYellow}*${NC}] WARNING: There may be bugs in the code."
-	echo -e "[${BYellow}*${NC}] WARNING: Please report them to the Pacstall team through \e]8;;https://github.com/pacstall/pacstall/issues\aGitHub\e]8;;\a or \e]8;;https://discord.com/invite/yzrjXJV6K8\aDiscord\e]8;;\a"
+fi
+
+echo -e "[${BGreen}+${NC}] INFO: updated from ${BGreen}${old_pacstall_version[0]}${NC} (${BGreen}${old_pacstall_branch[0]} ${old_pacstall_branch[1]}${NC}) -> ${BGreen}${new_pacstall_version[0]}${NC} (${BGreen}${new_pacstall_branch[0]} ${new_pacstall_branch[1]}${NC})"
+
+if [[ "${new_pacstall_branch[0]}" != "pacstall" ]] || [[ "${new_pacstall_branch[1]}" != "master" ]]; then
+	echo -e "[${BGreen}+${NC}] INFO: You have updated to a development branch."
+	echo -e "[${BYellow}*${NC}] WARN: Please remember that bugs may arise, and that this branch may not be as stable as master."
 
 fi
-echo "$USERNAME $BRANCH" | sudo tee "$STGDIR/repo/update" > /dev/null
+
+if [[ "${new_pacstall_branch[0]}" == "pacstall" ]]; then
+	echo -e "Useful links:"
+	echo -e "\t${BYellow}Website${NC}: ${BOLD}https://pacstall.dev${NORMAL}"
+	echo -e "\t${BPurple}Packages${NC}: ${BOLD}https://pacstall.dev/packages${NORMAL}"
+	echo -e "\t${BCyan}GitHub${NC}: ${BOLD}https://github.com/pacstall${NORMAL}"
+	if [[ "${new_pacstall_branch[1]}" != "master" ]]; then
```

Comment:
Why should this only be shown on non-master branches?

Reply:
Because it's safe to assume that most people updating to a development branch are using it for bug testing/new features

Reply:
Why shouldn't it be shown on master branch? Is the master branch free of all possible bugs, and therefore there isn't a need to report issues encountered there?